src/careamics/lightning/dataset_ng/lightning_modules/n2v_module.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any, Union
+
+from careamics.config import (
+    N2VAlgorithm,
+)
+from careamics.dataset_ng.dataset import ImageRegionData
+from careamics.lightning.dataset_ng.lightning_modules.unet_module import UnetModule
```

Comment:
Not important, but here the import is absolute, while it is relative in the `care_module`.